<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlLoginDlg</class>
 <widget class="QDialog" name="SqlLoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>63</y>
     <width>640</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgba(0,0,0,255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>640</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget
{
	background-color: rgba(170, 225,170,255);
	border-top-left-radius: 8px;
	border-top-right-radius: 8px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>8</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_quit
{
	border-image: url(:/icon/close_normal32.png);
}
QPushButton#btn_quit:hover
{
	border-image: url(:/icon/close_hover32.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>12</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: rgb(200, 60, 60);
}</string>
    </property>
    <property name="text">
     <string>数据库设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>64</y>
     <width>640</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
background-color: rgba(220, 255, 220,255);
}
QWidget#widget_2
{
border: 1px solid rgb(220,255,220);
border-bottom-left-radius: 8px;
border-bottom-right-radius: 8px;
}
</string>
   </property>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>89</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(220, 255, 220);
	border-radius: 20px;
	border: 1px solid rgba(0,0,0,0.5);
}
QPushButton:hover
{
	background-color: rgb(170, 225,170);
	border-radius: 20px;
	border: 1px solid rgba(0,0,0,0.5);
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24 9pt &quot;:/font/苹方黑体-细-简.ttf&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>端口(&amp;P)</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_port</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_user">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24 9pt &quot;:/font/苹方黑体-细-简.ttf&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>用户名(&amp;U)</string>
    </property>
    <property name="buddy">
     <cstring>Edit_user</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_pd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24 9pt &quot;:/font/苹方黑体-细-简.ttf&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>密码(&amp;M)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>Edit_pd</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_ip">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24 9pt &quot;:/font/苹方黑体-细-简.ttf&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>服务器地址(&amp;A)</string>
    </property>
    <property name="buddy">
     <cstring>Edit_ip</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>89</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(220, 255, 220);
	border-radius: 20px;
	border: 1px solid rgba(0,0,0,0.5);
}
QPushButton:hover
{
	background-color: rgb(170, 225,170);
	border-radius: 20px;
	border: 1px solid rgba(0,0,0,0.5);
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_ip">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>150</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit:hover
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>121.196.169.236</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="placeholderText">
     <string>xxx.xxx.xxx.xxx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_user">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>150</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit:hover
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>joky</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_pd">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>150</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit:hover
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>123456</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_port">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>150</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox:hover
{
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>3306</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>640</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_3
{
	background-color: rgb(220, 255, 220);
	border-radius: 8px;
}</string>
   </property>
  </widget>
  <zorder>widget_3</zorder>
  <zorder>line</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
 </widget>
 <tabstops>
  <tabstop>Edit_ip</tabstop>
  <tabstop>spinBox_port</tabstop>
  <tabstop>Edit_user</tabstop>
  <tabstop>Edit_pd</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btn_quit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
